### DISCUSSION TITLE
Problem creating a repository, the API is down?

### DISCUSSION THREAD

**Discussion by ssferraz** [community/community#53910] [Repositories]
### Select Topic Area

Bug

### Body

I am trying to create a repository today 04/27/2023 at 16:23, but the API that checks availability is loading infinitely. When opening the development tool of various browsers appears the error _"CAUGHET (in promise) Syntarseror: UnexPeded token '<'," <! Doctype "... is not valid json"_ and _"Failed to Load Resource: The Server Responsed With The status of 422 () "_

**Comment by dan296** [comment]
Yep everyone is having this issue currently.
